rudder-agent/SOURCES/debug-script/tests/rudder_unsent_inventories_agent

```diff
@@ -20,11 +20,19 @@ rudder_unsent_inventories_agent() {
   else
     UNSENT_INVENTORIES_DIR="${RUDDER_VAR}/inventories"
   fi
-
-  COMMAND_OUTPUT=`ls -l "${UNSENT_INVENTORIES_DIR}" 2>&1 | wc -l`
+  recent_time="432000"
+  for i in "${UNSENT_INVENTORIES_DIR}"
```

Comment:
This is a single path, there would be no reason to do a for on it.
Since it is a directory path I assume that you want to add a /* at the end

Reply:
changed.